F21/sujankandela/final_code (3).ipynb

```diff
@@ -0,0 +1,189 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 44,
+   "id": "102fd772",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
```

Comment:
good that you have two working program  . keep on optimize your code in a  single file